.changeset/many-pants-rule.md

```diff
@@ -0,0 +1,17 @@
+---
+'@twilio-paste/alert': patch
+'@twilio-paste/button': patch
+'@twilio-paste/card': patch
+'@twilio-paste/input': patch
+'@twilio-paste/list': patch
+'@twilio-paste/modal': patch
+'@twilio-paste/tooltip': patch
+'@twilio-paste/flex': patch
+'@twilio-paste/box': patch
+'@twilio-paste/sibling-box': patch
+'@twilio-paste/text': patch
+'@twilio-paste/theme': patch
+'@twilio-paste/core': patch
+---
+
+[Alert, Button, Card, Input, List, Modal, Tooltip, Flex, Box, SiblingBox, Text, Theme] Remove references to console theme in tests
```

Comment:
As an FYI you only need a changeset when you are touching code that is distributable. Tests and stories are internal assets, so they don't really need a changeset.

(I'm assuming my danger js script accommodates for that)